worker/codalabworker/docker_run.py

```diff
@@ -340,10 +346,11 @@ def _monitor(self):
                 report = False
             self._check_and_report_resource_utilization(report)
 
-            try:
-                self.resume()
-            except BundleServiceException:
-                pass
+            # Not sure what this was doing here...makes no sense.
+            #try:
+            #    self.resume()
+            #except BundleServiceException:
+            #    pass
```

Comment:
I believe this was introduced here. Was it causing problems? https://github.com/codalab/codalab-worksheets/pull/813/files#diff-a87348435a2a1245df564bae81d0df1cR268

Reply:
Yeah, it was causing a bug where a run would finish, the container would be deleted, and then the resume function would asynchronously try to finish the run again and it would fail because the container was gone.  Not sure why this didn't come up before.

Reply:
I think we should maybe get to the bottom of this a little? IIRC we actually had to [add similar code](https://github.com/semanticmachines/codalab-cli/pull/3/commits/202ebc13d73ebe6c56210c060be84e7462a11806#diff-0832a08fd6f7782ab5b63aef2c63d730R539) in the batch runner to prevent things from getting stuck in worker_offline forever.

Reply:
We talked offline; can you just add a comment here pointing to this PR?